scripts/00_Deploy_SwapKiwiV1.ts

```diff
@@ -0,0 +1,36 @@
+import { ethers } from 'hardhat';
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+
+const deployFunc: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  let contractOwnerAddress: string | undefined;
+
+  const initialAppFee = ethers.utils.parseEther(process.env.SWAP_KIWI_FEE ?? "0.0025");
+
+  if (hre.network.name === 'mainnet') {
```

Comment:
we should not deploy v1 contract anymore. Can you make it depend on some env variable like "DEPLOY_V1=true"?

Reply:
yeah, sure !!!